contracts/common/IVaultRecoverable.sol

```diff
@@ -6,6 +6,8 @@ pragma solidity ^0.4.24;
 
 
 interface IVaultRecoverable {
+    event RecoverToVault(address token, uint256 amount);
```

Comment:
Let's index the token address and also log the vault address where the tokens or eth are sent

Reply:
Also added the recovery vault as a parameter (since this could change in the org later).